crypto/fipsmodule/CMakeLists.txt

```diff
@@ -33,6 +33,45 @@ if(ARCH STREQUAL "x86_64")
     x86_64-mont5.${ASM_EXT}
     x86_64-mont.${ASM_EXT}
   )
+
+  if (UNIX AND NOT APPLE)
+    # Copy the s2n-bignum assembly files to the binary dir.
+    file(
+      COPY
+      ${PROJECT_SOURCE_DIR}/third_party/s2n-bignum/x86_att/p384/bignum_add_p384.S
```

Comment:
Might be void since I dunno if glob patterns can be applied here https://github.com/awslabs/aws-lc/pull/348/files#diff-07c678c6336e016855f80bdc854526fbce72ae60460690beeff0995bcc1a0e2dR55

Reply:
We can't use GLOB patterns for this because the s2n-bignum directory contains more files than we are using so we have to specify exactly which files to compile.

Reply:
I refactored the code to avoid duplication, please see the new commit.